### PR TITLE
Landingpage styling

### PR DESCRIPTION
**Pull request by vellip**
This PR styles the landing page for 1.5 Release:

- changed Hero Image style
- changed the style of project tiles (for a slider below the Hero Image)
- added a new highlight class for the Infoblock (purple)
- added an accent class for blocks (grey)
- added a columns block for three columns

### CODE REVIEW

**Comment by vellip** [comment]
@slomo Next would be to implement the block changes and outputting the projects carousel.

File: euth/projects/templates/euth_projects/includes/project_tile.html [hunk 1]
```diff
@@ -1,7 +1,13 @@
 {% load i18n thumbnail %}
-<div class="project-tile">
-    <a href="{% url 'project-detail' project.slug %}" class="project-tile-image-link"style="background-image: url('{{ project.image|thumbnail_url:'project_thumbnail' }}');">
-        <img src="{{ project.image|thumbnail_url:'project_thumbnail'}}" class="project-tile-image img-responsive"></a>
+<div class="project-tile {{ project.is_public|yesno:'public,private' }} {% if not project.image %} no-image {% endif %} {% if project.days_left <= 5 %} running-out {% endif %}">
+    <a href="{% url 'project-detail' project.slug %}" class="project-tile-image-link"
+       style="background-image: url('{% if project.image %} {{ project.image|thumbnail_url:'project_thumbnail' }} {% else %}/static/images/placeholder.png{% endif %}');">
```
Comment (slomo) [inline euth/projects/templates/euth_projects/includes/project_tile.html]: Loading files from static should be done via the`{% static '/images/placeholder.png'  %} tag. In order to use it you need to add `static` to `load`. More detail in the [django docs](https://docs.djangoproject.com/en/1.8/howto/static-files/).

File: projects/templates/projects/includes/project_tile.html [hunk 2]
```diff
@@ -1,9 +1,11 @@
 {% load i18n wagtailimages_tags wagtailcore_tags %}
 {% image project.image fill-800x400 as original %}
 <div class="project-tile style-{{project.projecttype|slugify}}">
+
```
Comment (slomo) [inline projects/templates/projects/includes/project_tile.html]: Some new whitespace for our whitespaces lovers :wink:  (not really critical)

**Comment by slomo** [comment]
Do you have any thought on how many items should go to the carousel?

**Comment by vellip** [comment]
I'd say eight, that'd be four visible immediately, two visible faded out and two can be seen by sliding, if any user ever uses that.

**Pull request merged**